src/serving.ts

```diff
@@ -291,4 +294,35 @@ export class Server {
       resolve();
     });
   }
+
+  verifyRecaptcha() {
+    return (req: express.Request, res: express.Response, next: express.NextFunction) => {
+      const recaptchaConfig = this.config.recaptchaConfig;
+
+      // Skip reCAPTCHA verification if the requisite config was not provided.
+      if (!recaptchaConfig) {
+        return next();
+      }
+
+      axios.default
+        .post('https://www.google.com/recaptcha/api/siteverify', {}, {
+          params: {
+            secret: recaptchaConfig.secretKey,
+            response: req.body.recaptchaToken,
+          }
+        })
+        .then((response: axios.AxiosResponse) => {
+          if (response.data['error-codes']) {
+            res.status(401).send('reCAPTCHA request failed: ' + response.data['error-codes']);
```

Comment:
Is there a library constant somewhere you can use for the 401 status?

Reply:
Good question. I found a library that defines those.